test/integration/framework.go

```diff
@@ -106,7 +107,7 @@ func getFreshApiserverAndClient(t *testing.T, storageTypeStr string) (servicecat
 	return clientset, shutdown
 }
 
-func waitForApiserverUp(serverIP string, stopCh <-chan struct{}) error {
+func waitForApiserverUp(insecureAddr string, stopCh <-chan struct{}) error {
```

Comment:
This is actually any URL. It's the whole thing cause all that happens is a straight `GET`. I was looking for an excuse to update it. I don't think setting up the transport hurt anything, but maybe it did.